src/Compilers/CSharp/Portable/Symbols/TypeWithState.cs

```diff
@@ -28,6 +29,44 @@ public static TypeWithState Create(TypeSymbol type, NullableFlowState defaultSta
             return new TypeWithState(type, state);
         }
 
+        public static TypeWithState Create(TypeWithAnnotations typeWithAnnotations, FlowAnalysisAnnotations annotations = FlowAnalysisAnnotations.None)
+        {
+            var type = typeWithAnnotations.Type;
+            if (type is null)
+            {
+                return new TypeWithState(null, NullableFlowState.MaybeNull);
```

Comment:
>return new TypeWithState(null, NullableFlowState.MaybeNull); [](start = 16, length = 60)

:memo: This line is not covered. Not sure if it could be.